### ISSUE TITLE
Generate valid file path from client without downloading the entire file

### ISSUE THREAD

**Issue by AndreasSchmid1** [saguiitay/WebDAVClient#14]
When I use the windows feature "map network drive..." I can directly access the files. Is this also possible with this library?

Thank you for the answer in advance!

**Comment by saguiitay** [comment]
I don't think that the WebDAV protocol allows accessing/downloading parts of files...

**Comment by AndreasSchmid1** [comment]
I tried the Windows feature "Map network drive..." and had the same "entire file downloading" problem. 	Wondrous I was able to read sections of the file with the NetDrive's Webdav feature.
